br/pkg/backup/client.go

```diff
@@ -282,15 +282,34 @@ func BuildBackupRangeAndSchema(
 	storage kv.Storage,
 	tableFilter filter.Filter,
 	backupTS uint64,
-) ([]rtree.Range, *Schemas, error) {
+	isFullBackup bool,
+) ([]rtree.Range, *Schemas, []*backuppb.Policy, error) {
```

Comment:
IMHO it's better to have a clear name "PlacementPolicy" compared with "Policy", say:
- `backuppb.Policy` -> `backuppb.PlacementPolicy`
- `(gs *tidbGlueSession) CreatePolicy(...` -> `(gs *tidbGlueSession) CreatePolicy(...`
...

Reply:
sure, PTAL https://github.com/pingcap/kvproto/pull/881/files